pennylane/_qubit_device.py

```diff
@@ -136,7 +136,7 @@ def _permute_wires(self, observable):
 
         .. code-block:: python
 
-              >>> print(0_1.wires)
```

Comment:
@antalszava 
 There is a minor typo here that I came across and thought to correct it.